libpod/container_exec.go

```diff
@@ -295,6 +299,10 @@ func (c *Container) ExecStart(sessionID string) error {
 
 	// Update and save session to reflect PID/running
 	session.PID = pid
+	session.PIDData, err = c.generatePIDData(pid)
```

Comment:
I'm a bit paranoid that if the process is already finished at this point, and someone else gets this PID number, the `PIDData` will contain data from someone else's process. I'd say the chances would be small, and maybe there's no need to address this.